.github/workflows/test.yml

```
@@ -93,6 +93,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      # This is a temporary solution.
+      # Once ubuntu-latest includes libxml2 >= v2.12.0, this step can be removed.
+      - name: Install latest libxml2 on PHP 8.4 (linux only)
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.php == '8.4' }}
+        run: |
+          /home/linuxbrew/.linuxbrew/bin/brew install libxml2
+
```

Per recent suggestion, let's add the version of this library before & after the install the new version to (a) verify that the install worked as expected, and (b) help us know when the image starts to contain the version that we need.
```suggestion
        run: |
          php -r 'echo "libxml version: ", LIBXML_DOTTED_VERSION, PHP_EOL;'
          /home/linuxbrew/.linuxbrew/bin/brew install libxml2
          php -r 'echo "libxml version: ", LIBXML_DOTTED_VERSION, PHP_EOL;'

```